tests/integration/test_hostgroup.py

```diff
@@ -0,0 +1,189 @@
+        await cluster.config_host_groups.create(
+            name=f"config-host-group-{i}", description=f"config host group description {i}"
+        )
+        config_host_groups = await cluster.config_host_groups.get(name__eq=f"config-host-group-{i}")
+        hosts = await cluster.hosts.filter(name__eq=f"test-host-{i}")
+        await config_host_groups.hosts.add(host=hosts)
+        await config_host_groups.hosts.add(host=await adcm_client.hosts.filter(name__eq=f"test-host-{109 - i}"))
+
+    await _test_confighostgroup_properties(cluster)
+    await _test_confighostgroup_accessors(cluster)
+    await _test_pagination_confighostgroup(cluster)
+
+
+async def _test_confighostgroup_properties(cluster: Cluster) -> None:
+    config_host_group = await cluster.config_host_groups.get(name__eq="config-host-group-30")
+    assert isinstance(config_host_group, ConfigHostGroup)
+    assert config_host_group.name == "config-host-group-30"
+    assert config_host_group.description == "config host group description 30"
+
+    hosts_all = await config_host_group.hosts.all()
+    assert len(hosts_all) == 2
+    assert isinstance(config_host_group.hosts, HostsInConfigHostGroupNode)
+
+
+async def _test_confighostgroup_accessors(cluster: Cluster) -> None:
+    assert not await cluster.config_host_groups.get_or_none(name__eq="config-host-group-300")
+    assert isinstance(await cluster.config_host_groups.get(name__eq="config-host-group-30"), ConfigHostGroup)
+
+    with pytest.raises(ObjectDoesNotExistError):
+        await cluster.config_host_groups.get(name__eq="fake_config-host-group")
+
+    with pytest.raises(MultipleObjectsReturnedError):
+        await cluster.config_host_groups.get(name__ne="fake_config-host-group")
+
+    assert len(await cluster.config_host_groups.all()) == 55
+
+    confighostgroup_list = await cluster.config_host_groups.list(query={"limit": 2, "offset": 1})
+    assert isinstance(confighostgroup_list, list)
+    assert len(confighostgroup_list) == 2
+
+    confighostgroup_list = await cluster.config_host_groups.list(query={"offset": 55})
+    assert isinstance(confighostgroup_list, list)
+    assert len(confighostgroup_list) == 0
+
+    async for c_h_group in cluster.config_host_groups.iter():
+        assert isinstance(c_h_group, ConfigHostGroup)
+
+    await (await cluster.config_host_groups.get(name__eq="config-host-group-30")).delete()
+    assert len(await cluster.config_host_groups.all()) == 54
+
+
+async def _test_actionhostgroup_properties(cluster: Cluster) -> None:
+    action_host_group = await cluster.action_host_groups.get(name__eq="action-host-group-35")
+    assert isinstance(action_host_group, ActionHostGroup)
+    assert action_host_group.name == "action-host-group-35"
+    assert action_host_group.description == "action host group description 35"
+
+    hosts_all = await action_host_group.hosts.all()
+    assert len(hosts_all) == 2
+    assert isinstance(action_host_group.hosts, HostsInActionHostGroupNode)
```

Comment:
since we don't have a separate task for that, `hosts` node should be tested for host groups: especially since we've had problems with paging in hosts of host groups

maybe it should be in a separate task

@a-alferov FYI

Reply:
Yes, let's add tests for the `hosts` node in the same task. @Starovoitov

Reply:
added